go/interpreter/lfvm/memory.go

```diff
@@ -68,68 +69,75 @@ func (m *Memory) ExpansionCosts(size uint64) tosca.Gas {
 	return fee
 }
 
-func (m *Memory) EnsureCapacity(offset, size uint64, c *context) error {
-	if size <= 0 {
-		return nil
-	}
+// expandMemoryAndCharge tries to expand memory to the given size.
+// If the memory is already large enough or size is 0, it does nothing.
```

Comment:
This seems to be not true any more. If `size` is `0` it will still grow the memory now if the `offset` is > `m.len()`.

Todo:
- check whether this property is important
- if so, add unit tests for this in `memory_test.go` and re-add support
- if not, remove this documentation

Reply:
this was changed as suggested, so the comment is still valid